src/applications/personalization/view-dependents/components/ViewDependentsFormStates/BuildFormlett.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ViewDependentsChildForm from './ViewDependentsChildForm.jsx';
+import ViewDependentsSpouseForm from './ViewDependentsSpouseForm.jsx';
```

Comment:
Don't think we need file extensions here.

Reply:
damn, Atom does this automatically and I dont always catch them